### PR TITLE
Topic/move core localization

### PR DESCRIPTION
**Pull request by DenRedsky**

### CODE REVIEW

File: demo/public/scripts/localization/localizationMap.ru.js [hunk 1]
```diff
@@ -89,12 +90,12 @@ var LANGMAPRU = {
     "CORE.FORM.VALIDATION.SYSTEMNAME" : "Системное имя содержит недопустимые символы.",
     "CORE.FORMATS" : "formats",
     "CORE.FORMATS.DATETIME" : "datetime",
-    "CORE.FORMATS.DATETIME.DAYSFULL" : "Воскресенье,Понедельник,Вторник,Среда,Четверг,Пятница,Суббота",
-    "CORE.FORMATS.DATETIME.DAYSSHORT" : "Вс,Пн,Вт,Ср,Чт,Пт,Сб",
+    "CORE.FORMATS.DATETIME.DAYSFULL" : "Sunday,Monday,Tuesday,Wednesday,Thursday,Friday,Saturday",
```
Comment (xSkara) [inline demo/public/scripts/localization/localizationMap.ru.js]: Something is wrong, the localization was valid before. The same problem in DE.

Comment (DenRedsky) [inline demo/public/scripts/localization/localizationMap.ru.js]: Fixed. This problem was caused by link deletion.

File: localization/.backup/2016-02-20_02_13_01_1cac2917f3744cd9b527ba47556f05e8/localization.de.n3 [hunk 2]
```diff
@@ -0,0 +1,114 @@
+﻿# <auto-generated>
```
Comment (xSkara) [inline localization/.backup/2016-02-20_02_13_01_1cac2917f3744cd9b527ba47556f05e8/localization.de.n3]: The `.backup` directory is a local artifact and should be removed from the repo.

Comment (DenRedsky) [inline localization/.backup/2016-02-20_02_13_01_1cac2917f3744cd9b527ba47556f05e8/localization.de.n3]: Fixed. Added in .gitignor.

**Pull request merged**